Sources/NyrisSDK/Enum/Public/AuthScope.swift

```diff
@@ -11,9 +11,7 @@ import Foundation
 public enum AuthScope : String {
     
     case imageMatching = "image_matching"
-    case hunterGatherer = "hunter_gatherer"
     case apiOfflineAccess = "api offline_access"
```

Comment:
api offline_access available only for internal usage

Reply:
ok, anything else ?